test/cbmc/sources/mqtt_cbmc_state.c

```diff
@@ -50,25 +50,19 @@
     #define REMAINING_LENGTH_MAX    CBMC_MAX_OBJECT_SIZE
 #endif
 
-void * mallocCanFail( size_t size )
-{
-    __CPROVER_assert( size < CBMC_MAX_OBJECT_SIZE, "mallocCanFail size is too big" );
```

Comment:
Can you confirm that this project is using all of the malloc flags, and that these malloc flags replicate this assertion failure being deleted here?